pkg/apis/servicecatalog/types.go

```diff
@@ -127,10 +127,9 @@ type ServiceClass struct {
 	Plans         []ServicePlan
 	PlanUpdatable bool // Do we support this?
 
-	// OSB-specific
-	// OSBGUID is the identity of this object for use with the OSB API.
+	// ExternalID is the identity of this object for use with the OSB API.
 	// Immutable.
-	OSBGUID string
+	ExternalID string
```

Comment:
We must have had a mismatch on slack.  I had expected that you would leave the `OSBGuid` field alone for ServiceClass and ServicePlan and we would transform these fields into the value of `Name` and add a new `DisplayName` field.  This is fine as is, and I can build upon this for changing those fields, but I'm glad I hadn't gotten to it yet (I had also forgotten when we talked on slack that I have bindable plans in progress and need to finish that first).

I'll pull this branch in and start on removing these fields for ServiceClass and ServicePlan, making the value of the OSB API ID for these types the value of the `Name` field, and adding a `DisplayName` field.